libs/application/templates/children-residence-change/src/fields/Overview/index.tsx

```diff
@@ -1,8 +1,10 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { useIntl } from 'react-intl'
+import { useMutation } from '@apollo/client'
 import { FieldBaseProps } from '@island.is/application/core'
-import { Box, Text, AlertMessage } from '@island.is/island-ui/core'
-import { DescriptionText } from '../components'
+import { Box, Text, AlertMessage, Button } from '@island.is/island-ui/core'
+import { CREATE_PDF_PRESIGNED_URL } from '@island.is/application/graphql'
+import { PdfTypes } from '@island.is/application/core'
```

Comment:
You already have the imported library 2 lines above

Reply:
🤦  Shouldn't the linter throw an error when this happens? 🤷

Reply:
I don't think so, because it's not wrong as is. We could probably enforce it with some eslint rules to avoid import duplication declarations